README.md

```diff
@@ -232,89 +392,121 @@ Available options:
+
+To help you navigate between the form fields here are the available hotkeys:
+
+| Key                               | Action                                                                               |
+|-----------------------------------|--------------------------------------------------------------------------------------|
+| <kbd>Esc</kbd>                    | Abort `summoner`                                                                     |
+| <kbd>Enter</kbd>                  | If input is valid go to the Confirm window                                           |
+| <kbd>Tab</kbd>                    | Change the focus to the next active field                                            |
+| <kbd>Shift</kbd> + <kbd>Tab</kbd> | Change the focus to the previous active field                                        |
+| <kbd>Space</kbd>                  | Enable/disable current checkbox or radio button                                      |
+| <kbd>Ctrl</kbd> + <kbd>d</kbd>    | Delete the input of the current text field                                           |
+| <kbd>Ctrl</kbd> + <kbd>u</kbd>    | Delete the input of the current text field from the cursor position to the beginning |
+| <kbd>Ctrl</kbd> + <kbd>k</kbd>    | Delete the input of the current text field from the cursor position to the end       |
+| <kbd>Ctrl</kbd> + <kbd>a</kbd>    | Move the cursor to the beginning of the current input field                          |
+| <kbd>Ctrl</kbd> + <kbd>e</kbd>    | Move the cursor to the end of the current input field                                |
+| <kbd>Up</kbd>/<kbd>Down</kbd>     | Scroll license list when focused                                                     |
+
+Note, that the form should be valid in order to be able to go to the Confirm window. If there are some errors in the input you could see the information about these errors in the `Error` section of the form.
+
+#### TUI show command [↑](#structure)
+
+These commands display the list of supported GHC versions or Licenses. Also, when the license name is specified the window with the scrollable content of the License text is shown.
+
+### CLI [↑](#structure)
 
-But when calling this command
+#### CLI new command [↑](#structure)
 
+In CLI mode of operation Summoner asks about every project setting. Most of the questions contain default values, so you can press <kbd>Enter</kbd> to choose a default value.
+If some option is specified via a configuration file or CLI arguments, then the question is skipped and the predefined value is used.
+
+#### CLI show command [↑](#structure)
+
+These commands display the list of the supported GHC versions, or Licenses. Also, when the license name is specified the content of the License is outputted to the terminal.
```

Comment:
list of ~~the~~ supported ..

when the license name is specified, the content of the license is ...

Reply:
```suggestion
These commands display the list of supported GHC versions, or Licenses. Also, when the license name is specified, the content of the License is outputted to the terminal.
```